vassal-app/src/main/java/VASSAL/build/module/PlayerRoster.java

```diff
@@ -493,14 +495,18 @@ public void finish() {
 
     // null is a cancel op - player will not connect to the game
     if (newSide != null) {
-      if (GameModule.getGameModule().isMultiplayerConnected()) {
-        final Command c = new Chatter.DisplayText(GameModule.getGameModule().getChatter(), Resources.getString(GlobalOptions.getInstance().chatterHTMLSupport() ? "PlayerRoster.joined_side_2" : "PlayerRoster.joined_side", GameModule.getGameModule().getPrefs().getValue(GameModule.REAL_NAME), translateSide(newSide)));
+      if (gm.isMultiplayerConnected()) {
+        final Command c = new Chatter.DisplayText(gm.getChatter(), Resources.getString(GlobalOptions.getInstance().chatterHTMLSupport() ? "PlayerRoster.joined_side_2" : "PlayerRoster.joined_side", gm.getPrefs().getValue(GameModule.REAL_NAME), translateSide(newSide)));
         c.execute();
+       // trying with logging in place of gm.getServer().sendToOthers(c);
```

Comment:
Looking good, but can you clean up these interim comments? 3 different places

Reply:
Done.